test/loadClient/go.mod

```
@@ -0,0 +1,9 @@
+module github.com/hazelcast/hazelcast-platform-operator/loadClient
+
+go 1.16
```

```suggestion
go 1.17
```
since the dockerfile use golang:1.17